docs/guides/modules/security/pages/audit-logs.adoc

```diff
@@ -1,22 +1,238 @@
 
+NOTE: Submitted dates are in UTC. To avoid issues arising from potential time zone differences, CircleCI adds extra time to the request. For example, if you request August 2 - August 5, the returned results will be in the range August 1 - August 6. The audit log request column is also displayed in UTC.
+
+[#request-status]
+=== Request status
+
+In the link:https://app.circleci.com/[CircleCI web app], a status will show the following information:
+
+- Timeframe requested
+- User who made the request.
+- Date request was made.
+- Expiry date of the request.
+- Request status.
+
+Statuses are updated once per hour on the 30-minute mark (for example, 09:30, 10:30, 11:30).
+
+Successful requests can be either of the following:
+
+- Active with a download link.
+- Active without any data (no download link).
+- Expired (no longer available to download).
+
+Once your request is successfully processed, select btn:[Download logs] to download the audit log as a CSV file.
+
+NOTE: Audit log fields with nested data contain JSON blobs.
+
+NOTE: Successful requests are available to download for 30 days.
+
+image::guides:ROOT:audit-log-request.png[Audit log requests]
+
+
+[#frequency-limits]
+=== Frequency limits
+
+NOTE: CircleCI may update the quantity and request period in the future.
+
+Frequency limits for requests are as follows:
+
+- Free Plan: 1 request per day
+- All paid plans: 3 requests per day
+- Maximum query window for 30 days - specifically, a customer can request logs for any 30 day window where the start of the window is within the last calendar year
+
+If your organization has reached the maximum amount of requests per 30 days, the audit log request feature will be disabled.
+
+If you hover over the disabled btn:[Request audit logs] button, a tooltip will appear displaying the date when new requests can be made. Pending requests count toward the rate limit.
+
+[#audit-log-streaming]
+== Audit Log Streaming
+
+NOTE: Even if you set up audit log streaming, you can still send xref:#audit-log-requests[audit logs requests] to CircleCI.
+
+The CircleCI Audit Log Streaming feature provides organizations with a solution for streaming audit logs directly to their own AWS S3 buckets.
+
+This functionality enables organizations to maintain comprehensive security visibility, meet compliance requirements, and integrate CircleCI audit data with their existing security monitoring infrastructure.
+
+[#prerequisites]
+=== Prerequisites
+
+[NOTE]
+====
+Before setting up audit log streaming, ensure you have the following:
+
+* CircleCI Scale plan (not available to Free or Performance customers)
+* AWS account with permissions to create and manage S3 buckets
+* IAM role or user with appropriate S3 write permissions
+* Network configuration to allow secure connections between CircleCI and AWS
+====
+
+The following steps describe how to set up AWS prerequisites:
```

Comment:
needs a title, for example === Set up AWS bucket and authentication (is that the right word?)